src/js/index.js

```diff
@@ -0,0 +1,130 @@
+// import { DateTime } from 'luxon';
```

Comment:
- [ ] Please kindly remove all the commented lines of code in your project, they will make your project look messy. 👍 **Note: this is applicable to line 36 - 130 as well**